app/transformers/api/v4/user_transformer.rb

```diff
@@ -4,12 +4,19 @@ def to_response(user)
       Api::V4::Transformer.to_response(user)
         .merge("registration_facility_id" => user.registration_facility.id,
                "phone_number" => user.phone_number,
-               "password_digest" => user.phone_number_authentication.password_digest)
+               "password_digest" => user.phone_number_authentication.password_digest,
+               "capabilities" => app_capabilities(user))
```

Comment:
How about `user.app_capabilities` instead? There's already tons of capability logic in the user model.

Reply:
I was slightly conflicted about this, since this is mostly transformation logic. (`true -> "yes", false -> "no"`). This is another variant I tried:

```
CAPABILITY_VALUES = {
 true => "yes",
 false => "no"
}.freeze

def app_capabilities(user)
  {can_teleconsult: User::CAPABILITY_VALUES[user.can_teleconsult?]}
end
```

Reply:
Actually this ^ + `user.app_capabilities` doesn't seem too bad.

Reply:
Can we simplify the data model of this capabilities blob? What about:

```
"user": {
  "capabilities": {
    "can_teleconsult": true
  }
}
```
or _even_ simpler
```
"user": {
  "capabilities": ["can_teleconsult"]
}
```

Reply:
1. The mobile devs prefer `"yes/no"` over `true/false` because it sits well with the other APIs (medical history has yes/no answers)
2. We considered this before and then moved to this hash map version since it's a bit more flexible (if we want to support more than yes/no types of `capabilities`).

Reply:
Gotcha. Then your suggestion above seems nice to me as well!